src/main/java/net/mcreator/ui/minecraft/states/block/JBlockPropertiesStatesList.java

```diff
@@ -123,6 +123,44 @@ private void createPropertiesEntry() {
 		}
 	}
 
+	private void addPropertyFromDataList() {

```

Comment:
For a bit more separation, I recommend moving this to AddBlockPropertyDialog and renaming current method from showDialogCustom, and adding new one showDialogExisting + test for that

Reply:
:heavy_check_mark: Fixed